proto/prysm/v1alpha1/attestation.go

```diff
@@ -181,6 +216,21 @@ func (a *AttestationElectra) Version() int {
 	return version.Electra
 }
 
+// IsNil --
+func (a *AttestationElectra) IsNil() error {
+	if a == nil {
+		return errors.New("nil AttestationElectra value")
+	}
+	if a.GetData() == nil {
+		return errors.New("attestation data is nil")
+	}
+	if a.GetData().Source == nil || a.GetData().Target == nil {
```

Comment:
will remove this from here as it breaks a lot of existing unit tests, will keep it in the validate functions